src/pages/VideoListing/VideoListing.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Sidebar } from '../../components';
+import { useData } from '../../contexts/data-context';
+import { VideoCard } from './components/VideoCard';
```

Comment:
can `import` directly from components since you defined an `index.js` file

Reply:
So this VideoCard is inside pages VideoListing components so because of that import like this